Kotlin/app/build.gradle

```diff
@@ -50,7 +50,7 @@ android {
 
 dependencies {
     // Import Taboola SDK
-    implementation 'com.taboola:android-sdk-beta:meta-unified-1.0.0-570-8bc74b127'
+    implementation 'com.taboola:android-sdk-beta:meta-twenty-minutes-beta-2-598-dea2d0cb0'
```

Comment:
@voltfmanSlava - Any reason we are using a specific version of a publisher here instead of a unified version?

Reply:
@tomer-br-taboola will update to the latest unified version.

Reply:
Done